### DISCUSSION TITLE
State value for SRTS-A01 stay on "heat"

### DISCUSSION THREAD

**Discussion by Gnol86** [Koenkk/zigbee2mqtt#14976] [Devices]
Hello,

First, thank you for the job.

My Aqara E1 Thermostat's state is always set to "heat". I think is appropriée to set it on "heat" then the valve il open and to "idle" then it is close.

<img width="527" alt="image" src="https://user-images.githubusercontent.com/24519929/201658171-69ddaefa-ab19-4b28-94ac-072428b4cfbd.png">

```
state climate.chambre=heat
    hvac_modes=['off', 'heat']
    min_temp=5.0
    max_temp=30.0
    target_temp_step=0.5
    preset_modes=['none', 'manual', 'away', 'auto']
    current_temperature=13.1
    temperature=10.0
    preset_mode=away
    away_preset_temperature=10.0
    battery=100
    calibrated=None
    calibration=True
    child_lock=LOCK
    linkquality=160
    local_temperature=13.1
    occupied_heating_setpoint=10
    preset=away
    sensor=internal
    system_mode=heat
    valve_detection=ON
    voltage=3300
    window_detection=ON
    window_open=None
    friendly_name=Chambre
    supported_features=17
```

**Comment by host-felice** [comment]
Did you understand when the valve is open/close?

**Reply by Gnol86** [reply]
Apparently, it’s not possible on this model 😔

**Reply by host-felice** [reply]
> Apparently, it’s not possible on this model 😔

Thank you for your reply! Did you understand when it starts to open? I mean, the difference between current and set temperature? And when it closes? So I can arrange a template to manage it.
